### PR TITLE
Remove karaf feature its unused and not up to date

### PR DESCRIPTION
**Pull request by ryeats**
#### What does this PR do?
The process of creating a distribution only pulls in transitives so we don't have to keep hand crafted versions of feature files like karaf, cxf, or camel anymore. This PR removes the karaf feature file because it is not being kept up to date with the latest releases of karaf and can cause outdated bundles to be installed.

#### Who is reviewing it? 
@KalCramer @coyotesqrl 

#### Ask 2 committers to review/merge the PR and tag them here.
@shaundmorris 
@rzwiefel

#### What are the relevant tickets?
Fixes: #____

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by ryeats** [comment]
build now

File: platform/security/rest/security-rest-cxfwrapper/src/main/java/org/codice/ddf/cxf/oauth/OAuthSecurityImpl.java [hunk 1]
```diff
@@ -326,7 +326,12 @@ private String getNewAccessToken(
     }
 
     if (response.getStatus() != HttpStatus.SC_OK) {
-      LOGGER.debug("Unable to retrieve system access token. {}", body);
+      LOGGER.debug(
```
Comment (shaundmorris) [inline platform/security/rest/security-rest-cxfwrapper/src/main/java/org/codice/ddf/cxf/oauth/OAuthSecurityImpl.java:329]: ❔ How does this relate to removing the karaf feature?

Comment (ryeats) [reply platform/security/rest/security-rest-cxfwrapper/src/main/java/org/codice/ddf/cxf/oauth/OAuthSecurityImpl.java:329]: oops didn't intend for that to be a part of this PR

**Comment by ryeats** [comment]
build now

**Pull request merged**